src/lib/api/api.go

```diff
@@ -119,8 +118,7 @@ func (a *API) Register(router *mux.Router) {
 func (a *API) Home(w http.ResponseWriter, r *http.Request) {
 	msg := "ping from " + r.RemoteAddr + ", sent pong in monitor"
 	log.Infoln(msg)
-	a.Data = msg
```

Comment:
I don't see `Data` field from *API removed yet. Could you please do that as per discussions?

Reply:
So should I remove the `Data` field from the *API struct and make the changes accordingly everywhere?

Reply:
Yes, that's my point from starting

Reply:
Okay

Reply:
Kindly review @Harkishen-Singh